jupyter-extension/src/plugins/mount/components/SortableList/__tests__/SortableList.test.tsx

```diff
@@ -501,4 +514,58 @@ describe('sortable list components', () => {
     fireEvent.change(getByTestId('ListItem__select'), {target: {value: 'dev'}});
     expect(getByTestId('ListItem__mount')).not.toBeDisabled();
   });
+
+  it('project filtering', async () => {
```

Comment:
I would recommend installing `eslint-plugin-testing-library` and `eslint-plugin-jest`. 

I did this recently for haberdashery; you can see how to do it in the first couple commits of:

1. https://github.com/pachyderm/haberdashery/pull/922
2. https://github.com/pachyderm/haberdashery/pull/923

Reply:
Thanks for the review!